pwnlib/dynelf.py

```diff
@@ -580,7 +580,7 @@ def lookup (self, symb = None, lib = None):
                 self.status("Trying remote lookup")
                 result = dynlib._lookup(symb)
         else:
-            result = dynlib.libbase
+            result = dynlib._lookup(symb)
```

Comment:
Hmm, this is incorrect, but I see why this looks right -- it's because a previous change to this file added the `and self.libcdb` to the above `if` statement.

We need to break up the logic in the above block.  This `else` statement used to be, effectively, `if not symb`.

Reply:
Ah yes, I forgot about that case, it looked more like a typo when I first spotted it. I'll update the docs for it as well.